### DISCUSSION TITLE
LucideIcon with Vue 3 Hover with tailwind

### DISCUSSION THREAD

**Discussion by EmmanuelMartinez** [lucide-icons/lucide#1429] [Q&A] [answered]
Hi, I'm using LucideIcon with Vue 3 and Tailwind.
Can I make the color change on hover? since it is svg with the color property
` 
import { XCircle } from 'lucide-vue-next';
<x-circle class="w-[24px]  hover:text-primary-accent"/>
`

**Comment by ericfennis** [comment, marked as answer]
@EmmanuelMartinez Yess this should work. Make sure you configured the colors in your tailwind config.
Here I've a [sandbox](https://codesandbox.io/p/github/ericfennis/lucide-vue-tailwind/main?layout=%257B%2522sidebarPanel%2522%253A%2522EXPLORER%2522%252C%2522rootPanelGroup%2522%253A%257B%2522direction%2522%253A%2522horizontal%2522%252C%2522contentType%2522%253A%2522UNKNOWN%2522%252C%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522id%2522%253A%2522ROOT_LAYOUT%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522UNKNOWN%2522%252C%2522direction%2522%253A%2522vertical%2522%252C%2522id%2522%253A%2522clk2i58qu000g3b6mdgsawkcw%2522%252C%2522sizes%2522%253A%255B70%252C30%255D%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522EDITOR%2522%252C%2522direction%2522%253A%2522horizontal%2522%252C%2522id%2522%253A%2522EDITOR%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL%2522%252C%2522contentType%2522%253A%2522EDITOR%2522%252C%2522id%2522%253A%2522clk2i58qu000b3b6mfqapdaal%2522%257D%255D%252C%2522sizes%2522%253A%255B100%255D%257D%252C%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522SHELLS%2522%252C%2522direction%2522%253A%2522horizontal%2522%252C%2522id%2522%253A%2522SHELLS%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL%2522%252C%2522contentType%2522%253A%2522SHELLS%2522%252C%2522id%2522%253A%2522clk2i58qu000f3b6m19qzsldt%2522%257D%255D%252C%2522sizes%2522%253A%255B100%255D%257D%255D%257D%252C%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522DEVTOOLS%2522%252C%2522direction%2522%253A%2522vertical%2522%252C%2522id%2522%253A%2522DEVTOOLS%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL%2522%252C%2522contentType%2522%253A%2522DEVTOOLS%2522%252C%2522id%2522%253A%2522clk2i58qu000d3b6m5x23y0hp%2522%257D%255D%252C%2522sizes%2522%253A%255B100%255D%257D%255D%252C%2522sizes%2522%253A%255B50%252C50%255D%257D%252C%2522tabbedPanels%2522%253A%257B%2522clk2i58qu000b3b6mfqapdaal%2522%253A%257B%2522id%2522%253A%2522clk2i58qu000b3b6mfqapdaal%2522%252C%2522activeTabId%2522%253A%2522clk2i5j8300xf3b6mnwmemtbt%2522%252C%2522tabs%2522%253A%255B%257B%2522id%2522%253A%2522clk2i58qt000a3b6m0q1h94cp%2522%252C%2522mode%2522%253A%2522permanent%2522%252C%2522type%2522%253A%2522FILE%2522%252C%2522filepath%2522%253A%2522%252FREADME.md%2522%252C%2522state%2522%253A%2522IDLE%2522%257D%252C%257B%2522type%2522%253A%2522FILE%2522%252C%2522filepath%2522%253A%2522%252Fsrc%252FApp.vue%2522%252C%2522id%2522%253A%2522clk2i5j8300xf3b6mnwmemtbt%2522%252C%2522mode%2522%253A%2522temporary%2522%252C%2522state%2522%253A%2522IDLE%2522%257D%255D%257D%252C%2522clk2i58qu000d3b6m5x23y0hp%2522%253A%257B%2522id%2522%253A%2522clk2i58qu000d3b6m5x23y0hp%2522%252C%2522tabs%2522%253A%255B%257B%2522type%2522%253A%2522TASK_PORT%2522%252C%2522taskId%2522%253A%2522dev%2522%252C%2522port%2522%253A5173%252C%2522id%2522%253A%2522clk2i5d7s00u53b6m180a2w1z%2522%252C%2522mode%2522%253A%2522permanent%2522%252C%2522path%2522%253A%2522%252F%2522%257D%255D%252C%2522activeTabId%2522%253A%2522clk2i5d7s00u53b6m180a2w1z%2522%257D%252C%2522clk2i58qu000f3b6m19qzsldt%2522%253A%257B%2522id%2522%253A%2522clk2i58qu000f3b6m19qzsldt%2522%252C%2522activeTabId%2522%253A%2522clk2i5bff00j33b6mx9i8ua3d%2522%252C%2522tabs%2522%253A%255B%257B%2522id%2522%253A%2522clk2i58qu000e3b6mnvgeddns%2522%252C%2522mode%2522%253A%2522permanent%2522%252C%2522type%2522%253A%2522TERMINAL%2522%252C%2522shellId%2522%253A%2522clk2i59z8000pfmjh0mhu5dlu%2522%257D%252C%257B%2522type%2522%253A%2522TASK_LOG%2522%252C%2522taskId%2522%253A%2522dev%2522%252C%2522id%2522%253A%2522clk2i5bff00j33b6mx9i8ua3d%2522%252C%2522mode%2522%253A%2522permanent%2522%257D%255D%257D%257D%252C%2522showDevtools%2522%253Atrue%252C%2522showShells%2522%253Atrue%252C%2522showSidebar%2522%253Atrue%252C%2522sidebarPanelSize%2522%253A15%257D) with this example. 
Good luck!
